### PR TITLE
fix for #957 - Missing upload button on the new issue creation page

### LINKED ISSUE #957: Missing upload button on the new issue creation page
It's only available on the comment form on existing issues and pull requests page.

### PR DESCRIPTION
**Pull request by Ricardo-Silva91**
Fixes #957 

Just changed the regular expression used to detect if the current page is an 'issues' page.

### CODE REVIEW

**Comment by bfred-it** [comment]
I think this will cause more issues on the other extensions. Do you see anything in the console? If you do, we might need a new `pageDetect.isNewIssue`

**Comment by Ricardo-Silva91** [comment]
hello again, I made the new `pageDetect.isNewIssue` and added it to content.js.

**Comment by bfred-it** [comment]
Good! But I don’t think there’s any gist to embed on the newIssue page

File: source/libs/page-detect.js [hunk 1]
```diff
@@ -47,6 +47,8 @@ export const isGist = () => location.hostname.startsWith('gist.') || location.pa
 
 export const isIssue = () => /^issues\/\d+/.test(getRepoPath());
 
+export const isNewIssue = () => /^issues\/new/.test(getRepoPath());
```
Comment (bfred-it) [inline source/libs/page-detect.js]: These are sorted by name, can you move it?

File: source/content.js [hunk 2]
```diff
@@ -183,6 +183,11 @@ function ajaxedPagesHandler() {
 		});
 	}
 
+	if (pageDetect.isNewIssue()) {
+		enableFeature(addUploadBtn);
+		enableFeature(embedGistInline);
```
Comment (bfred-it) [inline source/content.js]: I don’t think there’s any gist to embed on this page

**Comment by Ricardo-Silva91** [comment]
made the changes you requested, let me know if there's something else to edit.

**Comment by bfred-it** [comment]
Thanks @Ricardo-Silva91! 🎉

**Pull request merged**